build/CMakeHelpers/ParseCsProjForCsFiles.sh

```diff
@@ -7,7 +7,7 @@
 # CMakeLists.txt wrappers around .csproj files.
 
 cat *.csproj 2>/dev/null \
-    | egrep '<(Compile|SettingsRegistryDef) Include=".*\.cs"' \
+    | egrep '<(Compile|SettingsRegistryDef|MlosSettingsRegistryDefinition) Include=".*\.cs"' \
```

Comment:
We renamed the `Item` name used in`SettingsRegistry.csproj` files for external projects.  This allows us to reuse that script for both when using `FetchContent` `cmake` integration.